source/sass/components/_donation-amount.scss

```diff
@@ -8,6 +8,13 @@
   &--other {
     display: flex;
     flex-direction: row;
+
+    &--container {
```

Comment:
Added this so the input for monthly donations will remain the same size as the other donation choices on tablets and bigger screens